spec/components/layout_for_public_spec.rb

```diff
@@ -47,6 +47,18 @@ def component_name
     assert_select ".gem-c-layout-for-public .gem-c-layout-header", false
   end
 
+  it "can omit the feedback component" do
+    render_component(hide_feedback_form: true)
+
+    assert_select ".gem-c-layout-for-public .gem-c-feedback", false
+  end
```

Comment:
Could a test be added for when `hide_feedback_form` is false and/or not present?

Reply:
Done!